pkg/controllers/logs/controller.go

```diff
@@ -26,15 +26,15 @@ func NewLogsController(log *zap.Logger, logsProvider logs.LogsProvider, router *
 }
 
 func (controller *LogsController) FilterLogs(gctx *gin.Context) {
-
-	var params logs.Parameters
-	err := gctx.BindJSON(&params)
-	if err != nil {
-		gctx.JSON(http.StatusInternalServerError, gin.H{ //If error is not nil, an internal server error might have ocurred
-			"An error occurred": err,
-		})
-	}
-	logsList, err := controller.logsProvider.FilterLogs(params)
+	podname := gctx.Request.URL.Query().Get("podname")
```

Comment:
There is a method `c.Bind`. I wondering if we can use it, instead of manually parsing arguments. But if not, there is no parameter validation.